src/main/sf_hud.c

```diff
@@ -160,7 +274,63 @@ s32 func_800863C8(void) {
     return var_v1;
 }
 
+#ifdef IMPORT_DATA_PENDING
+void func_80086444(void) {
+    s32 i = 9;
+    s32 j;
+
+    u8* D_800D1C9C[] = { D_6001B80, D_60020D0 };
+    s32 D_800D1CA4[] = {
```

Comment:
For short arrays like these, I think it's better to have them on a single line. You can take off the final comma to stop the formatter from making them multi-line. Otherwise looks good.